manifests/role/app_host/prod.pp

```diff
@@ -13,4 +13,5 @@
   include nebula::profile::ruby
   include nebula::profile::nodejs
   include nebula::profile::named_instances
+  include nebula::profile::yarn
```

Comment:
I know we want this on bulleit-1, but do we also want it on bulleit-2, julep-1, juice, hatcher-apps-001 and macc-apps-001? If not, we should split out a new role just for bulleit-1.